charts/cinder-csi-plugin/templates/controllerplugin-deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
           image: "{{ .Values.csi.attacher.image.repository }}:{{ .Values.csi.attacher.image.tag }}"
           imagePullPolicy: {{ .Values.csi.attacher.image.pullPolicy }}
           args:
+            - "-v={{ .Values.logVerbosityLevel }}"
```

Comment:
it should be --v= .. right? does one dash also work?

Reply:
@ramineni both forms are allowed.

```
$ ./openstack-cloud-controller-manager --help
...
  -v, --v Level                                                                                                                                                                       
                number for the log level verbosity

$ ./cinder-csi-plugin --help
...
  -v, --v Level                          number for the log level verbosity

$ ./manila-csi-plugin --help
...
  -v, --v Level                          number for the log level verbosity
```

Reply:
Usually single-character flags are denoted with a single dash.

Reply:
ok thanks, then should be ok